languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -11774,7 +11774,47 @@ USA
             </rule>
         </rulegroup>
     </category>
+
+
     <category id='REMOVE_PRONOUN' name='Remoção de pronomes' type='style' tone_tags="povrem">
+
+
+        <!-- POR ELE SABER por saber -->
+        <rule id='NOMECOMUM_PRONOME_VERBINF_REM_PRON' name="Simplificar: Nome Comum + Pronome + Verbo Infinitivo → Omitir Pronome" type='style' tone_tags="povrem">
+            <!-- Created by Marco A.G.Pinto with Ricardo Joseh Lima suggestions, Portuguese rule 2023-07-08 (8-JUL-2023+) -->
+            <!--
+            O rei envia o mensageiro com rapidez por ele saber que o tempo é curto. → O rei envia o mensageiro com rapidez por saber que o tempo é curto.
+            Os reis enviam o mensageiro com rapidez por eles saberem que o tempo é curto. → Os reis enviam o mensageiro com rapidez por saberem que o tempo é curto.
+            -->
```

Comment:
Personally, I don't think we need these types of comments, as they don't add much new information. Your examples are already tagged and tested as `example`s, and we can check the authorship of specific rules with a simple `git blame`.